### PR TITLE
Adding shared_sessions boolean column to the client_config table. Add…

### PR DESCRIPTION
**Pull request by achievedaniel**
## Summary

Adding a shared sessions boolean for the auth oath project

## Related issue(s)

department-of-veterans-affairs/va.gov-team/75380

## Testing done

- [ x ] *New code is covered by unit tests*

## What areas of the site does it impact?
Client Config table adding another column.

## Acceptance criteria

- [ x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x ]  No error nor warning in the console.
- [x ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: db/migrate/20240213175114_add_shared_sessions_to_client_config.rb
Comment (dickdavis) [inline db/migrate/20240213175114_add_shared_sessions_to_client_config.rb:3]: 🗣️ I think this is good to go, since we **should** be using a version of [Postgres more recent than 10](https://github.com/ankane/strong_migrations?tab=readme-ov-file#adding-a-column-with-a-default-value), but I'm not sure what version we are actually running.

Comment (dickdavis) [reply db/migrate/20240213175114_add_shared_sessions_to_client_config.rb:3]: Probably a good idea to update the [client config docs](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Products/Sign-In%20Service/configuration/client_config.md) once we land this feature.

**Review by dickdavis — COMMENTED** [review]
LGTM, holding off on approval since I'm not fully aware of the work y'all are doing.

**Review by rileyanderson — DISMISSED** [review]
Looks good

**Review by bosawt — CHANGES_REQUESTED** [review]
You need to update the timestamp of the schema when changing it. Otherwise looks good

**Comment by achievedaniel** [comment]
Changed the timestamp in the Schema

**Review by bosawt — APPROVED** [review]

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**
